### ISSUE TITLE
phonebook-functions.php throws array_filter and invalid argument php warnings

### ISSUE THREAD

**Issue by derekjbernard** [UCF/Main-Site-Theme#160]
Example errors:

```[Fri Sep 07 08:27:37 2018] [error] [client 10.208.16.61] PHP Warning:  array_merge(): Argument #3 is not an array in /smcaweb/wordpress/wp-content/themes/Main-Site-Theme/includes/phonebook-functions.php on line 47, referer: https://www.ucf.edu/phonebook/?query=Amber+lee
[Fri Sep 07 08:27:37 2018] [error] [client 10.208.16.61] PHP Warning:  array_merge(): Argument #3 is not an array in /smcaweb/wordpress/wp-content/themes/Main-Site-Theme/includes/phonebook-functions.php on line 47, referer: https://www.ucf.edu/phonebook/?query=Amber+lee
[Fri Sep 07 08:57:26 2018] [error] [client 10.208.16.60] PHP Warning:  array_filter() expects parameter 1 to be array, object given in /smcaweb/wordpress/wp-content/themes/Main-Site-Theme/includes/phonebook-functions.php on line 60, referer: https://www.ucf.edu/phonebook/?query=36740
[Fri Sep 07 08:57:26 2018] [error] [client 10.208.16.60] PHP Warning:  Invalid argument supplied for foreach() in /smcaweb/wordpress/wp-content/themes/Main-Site-Theme/includes/phonebook-functions.php on line 62, referer: https://www.ucf.edu/phonebook/?query=36740```

**Comment by derekjbernard** [comment]
https://github.com/UCF/Main-Site-Theme/blob/51925107aa083a035518e6b6b4f8cedba637cb88/includes/phonebook-functions.php#L59-L69

**Comment by cjg89** [comment]
Should be resolved in v3.2.1.  Please re-open if these warnings persist after we push this tag out to production.
